ui/package.json

```diff
@@ -10,7 +10,7 @@
     "lint": "ng lint",
     "e2e": "ng e2e",
     "codegen-clean": "rm -rf src/generated",
-    "codegen": "npm run codegen-clean && java -jar libproject/swagger-codegen-cli.jar generate --lang typescript-angular --input-spec ../api/src/main/resources/workbench.yaml --output src/generated --additional-properties ngVersion=2"
+    "codegen": "npm run codegen-clean && java -jar ../libproject/swagger-codegen-cli.jar generate --lang typescript-angular --input-spec ../api/src/main/resources/workbench.yaml --output src/generated --additional-properties ngVersion=2"
```

Comment:
Worth noting: If we had ruby installed in our dev docker image, we could call `./project.rb swagger-regen` here to avoid duplicated code.

Reply:
Good point. Yes, I'd like to avoid duplicating the path.

I'll defer to next time we edit this stuff, if that's OK, since I'm less familiar with editing Docker images & would like to get back to notebook stuff.